### DISCUSSION TITLE
Composite deck - new SAF object

### DISCUSSION THREAD

**Discussion by MartinSalak** [StructuralAnalysisFormat/StructuralAnalysisFormat-Doc#269] [Ideas]
Hello SAF community,

we would like to start working on composite deck - new SAF object which describes slab created by profiled sheeting and concrete layer.
What should we not miss? Would you welcome a specific profiled sheets library or an element behavior in FEM?
Let us know!
![image](https://user-images.githubusercontent.com/89633938/211511095-55fe66b8-f1ea-44f8-aa2a-65180bec10d8.png)

**Comment by daranguiz-bbi** [comment]
Hi, 
Arcelor decks are welcomed
Concrete class should be on the structuralmaterial sheet 
What about studs and bar reinforcement? Extra tabs for them?
Propped /unpropped for beams and deck 
FEM behaviour for load transfer
